packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -192,7 +192,8 @@ $btn-sizeXSmall-lineHeight      : 1;
  * Sets the btn colour to site primary colour
  */
 
-.o-btn--primary {
+.o-btn--primary,
+.o-btn--icon.o-btn--primary.o-btn--sizeLarge {
```

Comment:
I saw you removed this style earlier, did you actually need it in the end?

Reply:
Oh, looks like I messed up with a rebase

Reply:
Oh no, this one is correct, I removed one about fill colour and background should be changed

Reply:
For regular primary button background colour changes to orange if size is large when for icon button it should stay blue